src/Tools/ExternalAccess/FSharp/DocumentSpan.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp
+{
+    /// <summary>
+    /// Represents a <see cref="TextSpan"/> location in a <see cref="Document"/>.
+    /// </summary>
+    public readonly struct DocumentSpan : IEquatable<DocumentSpan>
```

Comment:
>public [](start = 4, length = 6)

Shouldn't these be internal?

Reply:
➡️ Covered by https://github.com/dotnet/roslyn/pull/34945#discussion_r274990544